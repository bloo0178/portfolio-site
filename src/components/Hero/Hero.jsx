import React from "react";
import styles from "./Hero.module.scss";
import logo from "./images/logo.svg";
import semiCircle from "./images/semi_circle.svg";

class Hero extends React.Component {
  render() {
    return (
      <div className={styles.container}>
        <div className={styles.logo}>
          <img src={logo} alt="logo" width="100%" height="100%" />
        </div>

      </div>
    );
  }
}

export default Hero;
