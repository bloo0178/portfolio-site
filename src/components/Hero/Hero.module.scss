@import "../../common/_variables";

.container {
  //background-color: transparent;
  background-image: url("./images/bixby_1920x1080-EDITED.jpg");
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  padding-left: 10%;
  padding-right: 10%;
  text-align: center;

  h2 {
    font-style: italic;
  }
}
/* ----------- TRIANGLE ------------- */
.container:before {
  content:"";
    display: block;
    position: absolute;
    bottom: 0;
    width: 50%;
    left: 0;
    border-bottom: 100px solid $primary-background;
    border-right: 20rem solid transparent;
    -webkit-box-sizing: border-box;
    -moz-box-sizing: border-box;
    box-sizing: border-box;
}

.container:after {
  content:"";
    display: block;
    position: absolute;
    bottom: 0;
    width: 50%;
    right: 0;
    border-bottom: 100px solid $primary-background;
    border-left: 20rem solid transparent;
    -webkit-box-sizing: border-box;
    -moz-box-sizing: border-box;
    box-sizing: border-box;
}
/* ----------- END TRIANGLE ----------- */
.title {
  max-width: 30rem;
  font-style: bold;
  font-weight: bold;
}

