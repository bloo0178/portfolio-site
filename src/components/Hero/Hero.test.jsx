import React from "react";
import { create } from "react-test-renderer";
import Hero from "./Hero";

describe("Hero component", () => {
  test("it matches the snapshot", () => {
    const component = create(<Hero />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
