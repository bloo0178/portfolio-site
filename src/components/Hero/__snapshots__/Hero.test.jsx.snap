// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero component it matches the snapshot 1`] = `
<div
  className="container"
>
  <div
    className="title"
  >
    <h1>
      Jeff Bloom
    </h1>
    <h2>
      // Full Stack Dev
    </h2>
  </div>
</div>
`;
